metadata/base/metadata-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: metadata
+  name: metadata-service
```

Comment:
owning to `namePrefix: metadata-`, this service will be named `metadata-metadata-service`, so I think name it `service` is ok here, similar as above Deployment

Reply:
Done!